beacon_node/store/src/hot_cold_store.rs

```diff
@@ -2591,6 +2707,14 @@ pub fn migrate_database<E: EthSpec, Hot: ItemStore<E>, Cold: ItemStore<E>>(
     }) {
         let (block_root, state_root, slot) = maybe_tuple?;
 
+        // Delete block from hot DB
+        hot_db_ops.push(StoreOp::DeleteBlock(block_root));
+        // Write block to cold DB
+        let block = store
+            .get_hot_blinded_block(&block_root)?
+            .ok_or(HotColdDBError::MissingBlockToFreeze(block_root))?;
+        store.blinded_block_as_cold_kv_store_ops(&block_root, &block, &mut cold_db_ops)?;
```

Comment:
Does the state_root_iter yield the same block_root more than once for skipped slots? If yes we are doing extra work here